### PR TITLE
[Translation] Use TranslationInterface

### PR DESCRIPTION
**Pull request by GSadee**
| Q               | A
| --------------- | ---
| Bug fix?        | yes |
| New feature?    | no |
| BC breaks?      | yes |
| Related tickets | based on #8653 |
| License         | MIT |

### CODE REVIEW

File: src/Sylius/Component/Product/Model/ProductOptionInterface.php [hunk 1]
```diff
@@ -23,11 +23,20 @@
  */
 interface ProductOptionInterface extends
     CodeAwareInterface,
-    ProductOptionTranslationInterface,
     TimestampableInterface,
     TranslatableInterface
```
Comment (Zales0123) [inline src/Sylius/Component/Product/Model/ProductOptionInterface.php]: We should add `ResourceInterface` here

File: src/Sylius/Component/Shipping/Model/ShippingMethodInterface.php [hunk 2]
```diff
@@ -26,7 +26,6 @@
 interface ShippingMethodInterface extends
     ArchivableInterface,
     CodeAwareInterface,
-    ShippingMethodTranslationInterface,
     TimestampableInterface,
     ToggleableInterface,
     TranslatableInterface
```
Comment (Zales0123) [inline src/Sylius/Component/Shipping/Model/ShippingMethodInterface.php]: I suppose here also.

Comment (GSadee) [reply src/Sylius/Component/Shipping/Model/ShippingMethodInterface.php]: Done 👍

**Comment by pjedrzejewski** [comment]
Thanks Grzesiu!

**Pull request merged**